plan/relations_test.go

```diff
@@ -345,3 +346,116 @@ func TestRelations_Copy(t *testing.T) {
 		})
 	}
 }
+
+// FakeRel is a pretend relation that allows direct control of its direct output schema.
+type FakeRel struct {
```

Comment:
Since FakeRel is only used in this test, let's avoid exporting.

Reply:
Done.